modules/gdscript/gdscript_warning.h

```diff
@@ -88,6 +88,7 @@ class GDScriptWarning {
 		NATIVE_METHOD_OVERRIDE, // The script method overrides a native one, this may not work as intended.
 		GET_NODE_DEFAULT_WITHOUT_ONREADY, // A class variable uses `get_node()` (or the `$` notation) as its default value, but does not use the @onready annotation.
 		ONREADY_WITH_EXPORT, // The `@onready` annotation will set the value after `@export` which is likely not intended.
+		IMPLICIT_CONVERSION_CAUSES_COPY, // Implicit type conversion between array types resulting in array being passed by value
```

Comment:
I am not a native speaker and I don’t know which option is more correct: `IMPLICIT_CONVERSION_CAUSES_COPYING` or `IMPLICIT_CONVERSION_CAUSES_COPY`. If the latter, then the PR header must be updated.

Also the comment is outdated, the warning is not only about arrays (at least potentially).

Reply:
Both are valid English, I'd say the current wording fits better, id say it fits with `discarded` form above